### ISSUE TITLE
MISSING_VAR misreported on code

### ISSUE THREAD

**Issue by lmajano** [cflint/CFLint#539]
The following is valid syntax and misreported

```java
// Create HTTP object
var oHTTP = new HTTP(
	method 			= arguments.method,
	url 			= "#variables.serviceURL#/api/#variables.version#/#arguments.resource#",
	charset 		= "utf-8",
	timeout 		= arguments.timeout,
	throwOnError 	= false,
	redirect 		= true
);
```

**Comment by KamasamaK** [comment]
Probably a duplicate of #517

**Comment by lmajano** [comment]
Should I close?
